assembl/static2/js/app/components/common/formControlWithLabel.jsx

```diff
@@ -36,24 +61,31 @@ class FormControlWithLabel extends React.Component {
   };
 
   /* onBlur validation */
-  setValidationState() {
+  setValidationState = () => {
+    const { value, required } = this.props;
     let errorMessage = '';
     let validationState = null;
-    const valueSize = this.props.value ? this.props.value.length : 0;
-    if (this.props.required && valueSize === 0) {
+    const valueSize = value ? value.length : 0;
+    if (required && valueSize === 0) {
       errorMessage = I18n.t('error.required');
       validationState = 'error';
     }
 
     this.setState({ errorMessage: errorMessage, validationState: validationState });
-  }
+  };
+
+  getLabel = () => {
+    const { label, required } = this.props;
+    const placeHolder = required ? `${label}*` : label;
```

Comment:
you should probably remove the `const` and just return here

Reply:
```Return assignment should not contain assignment``` or did I get you wrong?

Reply:
`return required ? `${label}*` : label;` should be ok, no?